securedrop/tests/test_crypto_util.py

```diff
@@ -133,14 +133,29 @@ def test_basic_encrypt_then_decrypt_multiple_recipients(self):
 
         self.assertEqual(message, plaintext_)
 
-    def test_genrandomid(self):
-        id = crypto_util.genrandomid()
+    def verify_genrandomid(self, locale):
+        id = crypto_util.genrandomid(locale=locale)
         id_words = id.split()
 
         self.assertEqual(id, crypto_util.clean(id))
         self.assertEqual(len(id_words), crypto_util.DEFAULT_WORDS_IN_RANDOM_ID)
         for word in id_words:
-            self.assertIn(word, crypto_util.words)
+            self.assertIn(word, crypto_util._get_wordlist(locale))
+
+    def test_genrandomid(self):
```

Comment:
Solid tests :+1:. One thought: where possible let's name these tests to be more descriptive, e.g. `test_genrandomid_default_locale_is_en`. I like this approach because it makes for very clear natural documentation of the expected behavior of the code (even before you read the test).

Reply:
done